doc/user/rbac.md

```diff
@@ -69,40 +73,12 @@ export NAMESPACE=<namespace>
 
 ### RBAC with ClusterRole (create-crd=true)
 
-1. Create a ClusterRole:
```

Comment:
Can you keep this?

Instead of having "Quick setup" and "Production setup", maybe change to "Quick setup" and "Detailed setup".